controls/radfiledialogs/dialog-controls/radopenfiledialog.md

```diff
@@ -24,27 +24,53 @@ To show the dialog call its __ShowDialog__ method. If a valid file is opened whe
 
 #### __[C#] Example 1: Show a open file dialog__
 {{region cs-radfiledialogs-radopenfiledialog-0}}
-	RadOpenFileDialog openFileDialog = new RadOpenFileDialog();
-	openFileDialog.Owner = theHostWindowInstance;
-	openFileDialog.ShowDialog();
-	if (openFileDialog.DialogResult == true)
+	public partial class MainWindow : Window

```

Comment:
Why do you need such extended snippet? Is this something that came from the feedback form?

Also, if we want to improve this snippet, I think we can show a real user case. Usually, you open a dialog when a button is clicked. Instead of showing it on InitializeComponent() you can use a Button_Click event handler.